ctapipe/reco/regressor_classifier_base.py

```diff
@@ -295,6 +296,39 @@ def load(cls, path, cam_id_list, unit=1):
 
         return self
 
+    @staticmethod
+    def scale_features(cam_id_list, feature_list):
```

Comment:
Since it seems that we are going to use the `Pipeline` approach, I don't think this is still necessary.